azavea/pwd-fishway
# Implement styled-components

## Overview
We have styled-components in our project, but were using .css files. This PR ports over styles from the .css files and creates a new GlobalStyles component per the styled-components docs.

Connects #32

### Notes
- Things should look and operate mostly like they did before. We will eventually want to break out components like the `<Carousel />` in `About.js` to make styling more simple, but I kept things as they are for now.
- Spacing and headings may look a little different—this is because of the  `<GlobalStyles />`. The CSS there is mostly a reset copied over from blaster.js.
- The screensaver page wasn't full height, so in the `<GlobalStyles />` css I made some adjustments to fix this.

## Testing Instructions
* `git pull`
* Confirm that you can click anywhere on the screensaver page to enter.
* Confirm that carousel is working on About page.
* Confirm that See The Fishway page content is there.
* Confirm that navigation is working properly.
